lib/xcake/target/sugar.rb

```diff
@@ -1,11 +1,17 @@
 require "xcodeproj"
 
 module Xcake
+  def self.strip_heredoc(docs)
+    indent = docs.scan(/^[ \t]*(?=\S)/).min
+    indent_len = (indent || "").length
+    docs.gsub(/^[ \t]{#{indent_len}}/, "")
+  end
+
```

Comment:
Do you think we could put this in a separate file ? For example I have a method on Rubie's String class to escape it for Objective-C which lives in its own file :)